### PR TITLE
replaced artists with objects and other small text changes

### PR DESCRIPTION
**Pull request by corinnebosley**
I know that this is a bit of a controversial change, and some people will not be entirely happy about it, but I feel that the constant use of the mpl term 'artist' is both confusing and unnecessary.

While it is not actually wrong, I have never come across this term being used anywhere since I took this course myself, and the term 'object' is equally correct and more familiar to people who are new to Python.

All other changes are minor, and there are more PRs to come with slightly more significant changes.

### CODE REVIEW

**Comment by corinnebosley** [comment]
@dkirkham @pp-mo @dkillick 
Sorry about the huge diff, this is just because I removed all outputs so that it wasn't as massive.  Didn't work though, just made the diff massive.

**Comment by corinnebosley** [comment]
@djkirkham ?

File: course_content/notebooks/matplotlib_intro.ipynb [hunk 1]
```diff
@@ -27,49 +27,36 @@
...
-     "prompt_number": 2
+     "outputs": []
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "On its own, drawing the figure artist is uninteresting and will result in an empty piece of paper (that's why we didn't see anything above).\n",
-      "\n",
-      "By far the most useful artist in matplotlib is the \"Axes\" artist. The Axes artist represents the \"data space\" of a typical plot. A rectangular axes (the most common axes, but not the only axes, e.g. polar plots) will have two (confusingly named) Axis Artists with tick labels and tick marks.\n",
```
Comment (djkirkham) [inline course_content/notebooks/matplotlib_intro.ipynb]: "The Axes artist represents the \"data space\" of a typical plot"

Can we keep this line, replacing "artist" with "object"?

**Comment by djkirkham** [comment]
Looks good to me

**Comment by corinnebosley** [comment]
@pp-mo @dkillick @marqh Can somebody with merge rights please take a look at this so that I am not blocked for further changes?  Thank you!

File: course_content/notebooks/matplotlib_intro.ipynb [hunk 2]
```diff
@@ -81,25 +68,15 @@
...
-       "text": [
-        "<matplotlib.figure.Figure at 0x43c21d0>"
-       ]
-      }
-     ],
-     "prompt_number": 3
+     "outputs": []
```
Comment (pp-mo) [inline course_content/notebooks/matplotlib_intro.ipynb]: Did you intend to remove the output here ?
I think that this one is better left in, as a pre-loaded demonstration of the Figure.

I know that saving with output in some cells and the rest clear is pretty awkward, 
but it seems to be part of the original design intent.

In general, if we can control it, then I think leaving specific outputs in can make a notebook much more useful when you don't have a live version (e.g. when viewing in GitHub).
For example, "numpy_intro.ipynb" seems to need the outputs to make it readable; they are used as part of the discussion / explanations.

@corinnebosley @djkirkham **Do we feel this is generally necessary and/or worth it ?**

File: course_content/notebooks/matplotlib_intro.ipynb [hunk 3]
```diff
@@ -141,24 +118,14 @@
...
-       "text": [
-        "<matplotlib.figure.Figure at 0x43d4610>"
-       ]
-      }
-     ],
-     "prompt_number": 4
+     "outputs": []
```
Comment (pp-mo) [inline course_content/notebooks/matplotlib_intro.ipynb]: Output lost again here

File: course_content/notebooks/matplotlib_intro.ipynb [hunk 4]
```diff
@@ -27,49 +27,36 @@
...
      "cell_type": "markdown",
      "metadata": {},
      "source": [
       "## The matplotlib figure\n",
       "\n",
-      "At the heart of **every** matplotlib plot is the \"Figure\" object. The \"Figure\" object is the top level concept that can be drawn to one of the many output formats, or simply just to screen. Any object that can be drawn in this way is known as an \"Artist\" in matplotlib.\n",
-      "\n",
-      "Let's create our first artist using pyplot, and then show it:"
+      "At the heart of every matplotlib plot is the \"Figure\" object. The \"Figure\" object is the top level concept that can be drawn to one of the many output formats, or simply just to screen.\n",
+      "Lets create our first Figure object using pyplot, and then show it:"
```
Comment (corinnebosley) [inline course_content/notebooks/matplotlib_intro.ipynb]: Does it matter that Axes is technically a plural and I will be referring to it in the singular?  This is the main reason I have used the word object so much.

File: course_content/notebooks/matplotlib_intro.ipynb (hunk 2 above)
Comment (corinnebosley) [inline course_content/notebooks/matplotlib_intro.ipynb]: I think you may be right as far as the static notebooks are concerned.  I generally work to the principle that when writing notebooks and committing to source control, it is always best to clear all output for size and simplicity purposes, but maybe that's not the case here.

Would it be better to create the static notebooks for the Readme with cell outputs, but have clear outputs for the live notebooks.  After all, in the course itself, all the notebooks' cells will be executed in order.

File: course_content/notebooks/matplotlib_intro.ipynb [hunk 5]
```diff
@@ -27,49 +27,36 @@
...
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "On its own, drawing the figure artist is uninteresting and will result in an empty piece of paper (that's why we didn't see anything above).\n",
-      "\n",
-      "By far the most useful artist in matplotlib is the \"Axes\" artist. The Axes artist represents the \"data space\" of a typical plot. A rectangular axes (the most common axes, but not the only axes, e.g. polar plots) will have two (confusingly named) Axis Artists with tick labels and tick marks.\n",
-      "\n",
-      "There is no limit on the number of Axes artists that can exist on a Figure artist. Let's go ahead and create a figure with a single Axes Artist, and show it using pyplot:"
+      "On its own, drawing the figure is uninteresting and will result in an empty piece of paper (that's why we didn't see anything above).\n",
+      "But we can draw an Axes object in this figure which represents our data space. The most basic Axes object is rectangular and will have tick labels and tick marks. There is no limit on the number of Axes objects that can exist on a Figure object.\n",
```
Comment (corinnebosley) [inline course_content/notebooks/matplotlib_intro.ipynb]: Agreed.

File: course_content/notebooks/matplotlib_intro.ipynb [hunk 6]
```diff
@@ -27,49 +27,36 @@
...
+      "Let's go ahead and create a figure with a single Axes object, and show it using pyplot:"
```
Comment (corinnebosley) [inline course_content/notebooks/matplotlib_intro.ipynb]: Agreed.

File: course_content/notebooks/matplotlib_intro.ipynb (hunk 2 above)
Comment (corinnebosley) [inline course_content/notebooks/matplotlib_intro.ipynb]: @djkirkham @pp-mo @dkillick I think we need to agree on a strategy here before I start chopping cell outputs in and out of the commits.

**Comment by corinnebosley** [comment]
@pp-mo I think we're set then.  I have made the changes you suggested and replaced the outputs.  Let's get this merged (as soon as you have checked it over, obviously).

File: course_content/notebooks/matplotlib_intro.ipynb (hunk 4 above)
Comment (pp-mo) [inline course_content/notebooks/matplotlib_intro.ipynb]: Well, "an Axes" is perfectly correct, it's just a bit awkward.
Though not as awkward as the Axes/Axis confusion in my opinion !

**Comment by corinnebosley** [comment]
@pp-mo @dkillick What are your thoughts on this attempt then?

**Comment by corinnebosley** [comment]
@dkillick updated

**Comment by pp-mo** [comment]
All done, except glitch noted in  https://github.com/SciTools/courses/pull/45#commitcomment-20661163

**Comment by corinnebosley** [comment]
@pp-mo @dkillick Please can somebody merge this now? Please...?

**Comment by dkillick** [comment]
I'm doing it!!

**Comment by dkillick** [comment]
:tada: :taco: :100:

**Comment by dkillick** [comment]
Nice work @corinnebosley @pp-mo

**Comment by corinnebosley** [comment]
Thank you!

**Pull request merged**